.github/workflows/pull-request-checks.yaml

```diff
@@ -39,6 +39,7 @@ jobs:
           make -C regression/cbmc test-paths-lifo
           env PATH=$PATH:`pwd`/src/solvers make -C regression/cbmc test-cprover-smt2
           make -C jbmc/regression test
+
```

Comment:
Why is there a commit that's adding something called a "draft" in here if this is no longer a draft PR? It also shouldn't add this empty line here.

SHA: cfdcdca5b7091d0ea18c918f8f1159c4ef217b61

Reply:
Right, but now that you've done all that I don't see a good reason not to consolidate these changes. Also, this doesn't change the fact there's a spurious newline here.

Reply:
I much prefer they are not consolidated - this was fleshed out in bits, and I prefer that the history reflects that.

For the empty line, it looks spurious, I agree, but this was introduced because without it the file looks badly formatted - there's no new line between two workflows, and at least to me, that looks weird, especially since it's inconsistent with the others.